README.md

```diff
@@ -64,11 +37,6 @@ In this course, you will:
 
 <footer>
 
-<!--
-  <<< Author notes: Footer >>>
-  Add a link to get support, GitHub status page, code of conduct, license link.
--->
-
 ---
 
 Get help: [Post in our discussion board](https://github.com/orgs/skills/discussions/categories/introduction-to-secret-scanning) &bull; [Review the GitHub status page](https://www.githubstatus.com/)
```

Comment:
I notice that line 76 has a copyright date of 2023, which I also saw on the actions course that I ran this morning. Will we be able to automate this, or do we need a reminder to update this once a year throughout the Skills organization?